core/services/feeds/service.go

```diff
@@ -23,8 +25,16 @@ import (
 var (
 	ErrOCRDisabled        = errors.New("ocr is disabled")
 	ErrSingleFeedsManager = errors.New("only a single feeds manager is supported")
+
+	promJobProposalRequest = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
You only seem to be using this for `Inc` in which case I'd say you want a `Counter` and not a `Gauge`.

Reply:
Done